src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -315,6 +316,14 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         const auto selectedItem{ e.AddedItems().GetAt(0) };
         const auto newFontFace{ unbox_value<Editor::Font>(selectedItem) };
         Appearance().FontFace(newFontFace.LocalizedName());
+        if (!UsingMonospaceFont())
+        {
+            ShowProportionalFontWarning(true);
+        }
+        else
+        {
+            ShowProportionalFontWarning(false);
+        }
```

Comment:
nit: this can be simplified to `ShowProportionalFontWarning(!UsingMonospaceFont())`

Reply:
Ohhh. You're right. Cool 😎